git-cms-addpkg

```diff
@@ -70,23 +74,40 @@ while [ "$#" != 0 ]; do
       INITOPTIONS="$INITOPTIONS $1"
       shift;;
     -*)
-      echo Unknown option $1 ; usage 1 ;;
+      $ECHO "git cms-addpkg: unknown option $1"; $ECHO; usage 1;;
     *)
       if [ "$INPUT_FILE" == "" ]; then
         # check out a list of packages
         PACKAGES="$PACKAGES $1"
         shift
       else
         # check out a list of packages via -f / --file FILE
-        echo "git cms-addpkg: you cannot specify a package and input from file at the same time."
-        exit 1
+        $ECHO "git cms-addpkg: you cannot specify a package and input from file at the same time."
+        $ECHO
+        usage 1
       fi
     ;;
   esac
 done
-
 if [ "$PACKAGES" == "" ] && [ "$INPUT_FILE" == "" ] ; then
-  echo "You need to specify at least one package or input file." ; exit 1
+  $ECHO "git cms-addpkg: you need to specify at least one package or input file."
+  $ECHO
+  usage 1
+fi
+
+if [ $VERBOSE == 0 ]; then
+  # send vrbose messages to /dev/null
+  exec {verbose}> /dev/null
```

Comment:
from `man bash` and https://www.gnu.org/software/bash/manual/bash.html#Redirections
> Each redirection that may be preceded by a file descriptor number may instead be preceded by a word of the form {varname}.  In this case, for each  redirection  operator  except >&-  and  <&-,  the  shell  will  allocate  a file descriptor greater than or equal to 10 and assign it to varname.  If >&- or <&- is preceded by {varname}, the value of varname defines the file descriptor to close.

The use of `exec` for redirections is documented in many online articles, e.g. http://www.linuxjournal.com/content/bash-redirections-using-exec .